Gruntfile.js

```diff
@@ -194,6 +194,7 @@ module.exports = function (grunt) {
           styles: ['node_modules/datatables.net-dt/css/jquery.dataTables.css',
             'node_modules/patternfly/dist/css/patternfly.css',
             'node_modules/patternfly/dist/css/patternfly-additions.css',
+            'http://rawgit.com/junezhang/showcase/master/dist/styles/patternfly-showcase.css',
```

Comment:
The README file needs some work which will then need to be incorporated into this. :(
I am working on updating it.